SickDesignSystem/synergy
# chore: 🔧 Adjust components build so it exports all components automatically

# Pull Request

## 📖 Description

As we already forgot multiple times to export the created components in `src/synergy.ts` and our framework wrappers rely on the exports in this file, we have to make sure the exports are added without manual resolution. This PR adds the changes in the build that makes this possible.

### 🎫 Issues

Closes #146 

## 👩‍💻 Reviewer Notes

The main changes where done in `packages/components/scripts/jobs/createExports.js`.

## 📑 Test Plan

1. Check out this branch
2. Make sure dependencies are up to date (via `pnpm i -r`)
3. Go to `packages/components` and add a component to `scripts/config.js`
4. Vendor the new component
5. Issue a new build via `pnpm build`
6. Expection: The file `src/synergy.ts` should include a new export that is named after the newly included component.

## ✅ Checklist
